src/install.js

```diff
@@ -33,6 +33,15 @@ const handlebars = require('handlebars');
 var success = true;
 const argvNamespace = typeof (argv.n || argv.namespace) === 'string' ? argv.n || argv.namespace : 'razeedeploy';
 
+handlebars.registerHelper('list', function(items, options) {
+  var out = '';
+  for(let i = 0; i < items.length; i++){
+    out = out + options.fn(items[i]);
+  }
+  log.info(`list output is ${out}`);
+  return out;
+});
+
```

Comment:
is this necessary. i know mustache already has this as a built in help. I would assume handlebars would too since it is a superset of mustache

Reply:
see https://mustache.github.io/mustache.5.html at `Non-Empty Lists`

```
If the person key exists and has a non-false value, the HTML between the pound and slash will be rendered and displayed one or more times.

When the value is a non-empty list, the text in the block will be displayed once for each item in the list. The context of the block will be set to the current item for each iteration. In this way we can loop over collections.

Template:

{{#repo}}
  <b>{{name}}</b>
{{/repo}}
Hash:

{
  "repo": [
    { "name": "resque" },
    { "name": "hub" },
    { "name": "rip" }
  ]
}
Output:

<b>resque</b>
<b>hub</b>
<b>rip</b>
```

Reply:
you are right. removed list helper.